tests/infer/test_valid_models.py

```diff
@@ -1300,6 +1301,7 @@ def guide():
 
 
 @pytest.mark.parametrize('markov', [False, True])
+@pytest.mark.skipif("CUDA_TEST" in os.environ, reason="https://github.com/uber/pyro/issues/1380")
```

Comment:
This issue is already resolved. If this is still failing, try changing `range(20)` to `range(15)`.

Reply:
I had to use`range(11)` - anything above that fails.

Reply:
Oh I see, that makes sense since each time step allocates a new dim, so `range(11)` should allocate 22 dims.